src/Compilers/Core/Portable/Operations/OperationFactory.cs

```diff
@@ -62,7 +62,7 @@ public static IInvalidOperation CreateInvalidExpression(SemanticModel semanticMo
             return new InvalidOperation(children, semanticModel, syntax, type: null, constantValue: default(Optional<object>), isImplicit: isImplicit);
         }
 
-        public static Lazy<IOperation> NullOperation { get; } = new Lazy<IOperation>(() => null);
```

Comment:
>new Lazy<IOperation>(() => null); [](start = 64, length = 33)

Case in point: the actual changes related to Lazy are drowned out by mechanical renames :-(